OpenBudgeteer.Blazor/Shared/BucketDetailsDialog.razor

```diff
@@ -49,7 +50,14 @@
                                                 <td class="d-none d-md-table-cell">@transaction.SelectedAccount.Name</td>
                                                 <td class="d-none d-md-table-cell">@transaction.Payee</td>
                                                 <td>@transaction.Memo</td>
-                                                <td style="text-align: right">@transaction.Amount.ToString("C", CultureInfo.CurrentCulture)</td>
+                                                @if (transaction.Buckets.Count == 0)
+                                                {
+                                                    <td style="text-align: right">@transaction.Amount.ToString("C", CultureInfo.CurrentCulture)</td>
+                                                }
+                                                else
+                                                {
+                                                    <td style="text-align: right">@transaction.Buckets.Where(b => b.SelectedBucketId == DataContext.BucketId).Sum(b => b.Amount).ToString("C", CultureInfo.CurrentCulture)</td>
```

Comment:
Maybe we should add some kind of indication that this transaction is split between multiple buckets, like a simple `*`?

Reply:
Good idea